www/_blog/2022-02-22-supabase-beta-january-2022.mdx

```diff
@@ -180,14 +180,13 @@ Check out the video below.
 
 - Arctype published a new guide showing how to connect to your Supabase database [https://arctype.com/postgres/connect/supabase-postgres](https://arctype.com/postgres/connect/supabase-postgres)
 - https://github.com/marmelab/ra-supabase, Supabase adapter for react-admin, the frontend framework for building admin applications on top of REST/GraphQL services.
-- [~~https://hotgluexyz.notion.site/Supabase-and-hotglue-article-9e7f5583d27c419490ee7d536d6d269d](https://www.notion.so/Supabase-and-hotglue-article-9e7f5583d27c419490ee7d536d6d269d) (we can give them feedback on when we want this to be released)~~
 
 ### PostgREST
 
 One of Supabase’s key tools that allow a lot of our functionality has had some updates.
 
-- pg_listen was removed in favor of PostgREST’s built-in schema reloading https://github.com/supabase/infrastructure/issues/1531
```

Comment:
retaining the link is fine

Reply:
@darora Though people that don't have access to the infrastructure won't be able to see it.

Reply:
d'oh! right you are-let's remove it, sorry